sips/fork_support.md

```diff
@@ -91,14 +91,14 @@ Along with the `DomainType`, we include a `NetworkID` type and a new `ForkData`
 
 ```go
 // NetworkID are intended to separate different SSV networks. A network can have many forks in it.
-type NetworkID byte
-
-const (
-	MainnetNetworkID = NetworkID(0x0)
-	PrimusNetworkID  = NetworkID(0x1)
-	ShifuNetworkID   = NetworkID(0x2)
-	JatoNetworkID    = NetworkID(0x3)
-	JatoV2NetworkID    = NetworkID(0x4)
+type NetworkID [1]byte
+
+var (
+	MainnetNetworkID = NetworkID{0x0}
+	PrimusNetworkID  = NetworkID{0x1}
+	ShifuNetworkID   = NetworkID{0x2}
+	JatoNetworkID    = NetworkID{0x3}
+	JatoV2NetworkID  = NetworkID{0x4}
```

Comment:
I have no issues with this change.
I am just wondering why you decided to do it?  Seems like an implementation detail.

Reply:
It's an implementation detail indeed. I made the change due to marshaling problems.